app/[eventSlug]/proposals/proposal-table.tsx

```diff
@@ -233,14 +417,143 @@ export function ProposalTable({
+          >
+            <div className="space-y-3">
+              <div>
+                <h3 className="text-base font-medium text-gray-900">
+                  {proposal.title}
+                </h3>
+                <p className="text-sm text-gray-500 mt-1">
+                  Host(s):{" "}
+                  {proposal.hosts
+                    .map(
+                      (host) =>
+                        guests.find((g) => g.ID === host)?.Name || "Deleted"
+                    )
+                    .join(", ") || "-"}
+                </p>
+              </div>
+
+              {proposal.description ? (
+                <p className="text-sm text-gray-600 line-clamp-3">
+                  {proposal.description}
+                </p>
+              ) : (
+                <p className="text-sm text-gray-500">-</p>
+              )}
+
+              {proposal.durationMinutes ? (
+                <div className="flex items-center">
+                  <ClockIcon className="h-4 w-4 mr-1 text-gray-400" />
+                  <span className="text-sm text-gray-500">
+                    {formatDuration(proposal.durationMinutes)}
+                  </span>
+                </div>
+              ) : (
+                <div className="text-sm text-gray-500">-</div>
+              )}
+
+              <div className="pt-2 border-t border-gray-100 space-y-3">
+                <div className="flex gap-1">
+                  <div className="relative inline-block group">
+                    <button
+                      type="button"
+                      className="opacity-50 cursor-not-allowed rounded-md border border-black shadow-sm px-2 py-1 bg-white font-medium text-black focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-200 grayscale"
+                      disabled
+                    >
+                      ❤️
+                    </button>
+                    <div className="absolute bottom-full left-1/2 transform -translate-x-1/2 mb-2 px-2 py-1 text-sm text-white bg-gray-800 rounded opacity-0 group-hover:opacity-100 transition-opacity duration-200 pointer-events-none whitespace-nowrap z-10">
+                      Voting is not yet enabled
+                    </div>
+                  </div>
+                  <div className="relative inline-block group">
+                    <button
+                      type="button"
+                      className="opacity-50 cursor-not-allowed rounded-md border border-black shadow-sm px-2 py-1 bg-white font-medium text-black focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-200 grayscale"
+                      disabled
+                    >
+                      ⭐
+                    </button>
+                    <div className="absolute bottom-full left-1/2 transform -translate-x-1/2 mb-2 px-2 py-1 text-sm text-white bg-gray-800 rounded opacity-0 group-hover:opacity-100 transition-opacity duration-200 pointer-events-none whitespace-nowrap z-10">
```

Comment:
All the repeated divs should be a separate component 

```
const VotingDisabled = () => {
    return (
        <div className="absolute bottom-full left-1/2 transform -translate-x-1/2 mb-2 px-2 py-1 text-sm text-white bg-gray-800 rounded opacity-0 group-hover:opacity-100 transition-opacity duration-200 pointer-events-none whitespace-nowrap z-10">
             Voting is not yet enabled
        </div>
    )
}
```

Reply:
(The component should probably be in a separate file so it can be also reused in the proposal-table)